bundles/org.openhab.binding.mercedesme/src/main/java/org/openhab/binding/mercedesme/internal/AccountConfiguration.java

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
This class could be moved into a **config** subdirectory (often structured like this in other bindings).

Reply:
Moved